saleor/graphql/views.py

```diff
@@ -209,7 +207,7 @@ def parse_query(
     def execute_graphql_request(self, request: HttpRequest, data: dict):
         with opentracing.global_tracer().start_active_span("graphql_query") as scope:
             span = scope.span
-            span.set_tag(opentracing.tags.COMPONENT, "GraphQL")
+            span.set_tag(opentracing.tags.COMPONENT, "graphql")
```

Comment:
Why change this?

Reply:
I would expect the component having the same name as used in tags

Reply:
They are unrelated though?